tests/test_bgp_af.rb

```diff
@@ -282,9 +408,9 @@ def test_additional_paths
     # Test getter
     #
 
-    assert(bgp_af.additional_paths_send)
-    assert(bgp_af.additional_paths_receive)
-    assert(bgp_af.additional_paths_install)
+    assert(bgp_af.additional_paths_send, "No additional-paths send configured")
+    assert(bgp_af.additional_paths_receive, "No additional-paths receive configured")
+    assert(bgp_af.additional_paths_install) if platform == :nexus
```

Comment:
Is this property unsupported on XR? If so then I'd like to see something like this in the test:

```
if platform == :nexus
  assert(bgp_af.additional_paths_install)
elsif platform == :ios_xr
  assert_nil(bgp_af.additional_paths_install)
end
```